ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -656,7 +666,7 @@ describe('RuntimeConfigurationPanel', () => {
           runtimePresets.generalAnalysis.runtimeTemplate.gceConfig.machineType
         ).memory
       );
-      expect(getMainDiskSize(wrapper)).toEqual(
+      expect(getDetachableDiskSize(wrapper)).toEqual(
         runtimePresets.generalAnalysis.runtimeTemplate.gceConfig.diskSize
```

Comment:
these should be gceWithPdConfig, I think

Reply:
Fixed